daisy_workflows/image_import/debian/translate.py

```diff
@@ -49,8 +48,8 @@ def DistroSpecific(g):
   if install_gce == 'true':
     logging.info('Installing GCE packages.')
 
-    g.command(['apt-get', 'update'])
-    g.sh('DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes gnupg')
+    utils.update_apt(g)
+    utils.install_apt_package(g, 'gnupg')
```

Comment:
Probably minor: utils.install_apt_package uses --no-install-recommends flag while the replaced call above doesn't.

Reply:
I checked and it's being used by apt-key, and apt-key only requires gnupg.

To be confident, I added a test for Debian 8, so we have full coverage of the supported debian versions now.

Reply:
My comment was about the --no-install-recommends  being provided now by default in install_apt_package while some of the calls that were replaced by it didn't have the same flag. Will the behavior be the same?

Reply:
So, what I was trying to say is that the installation of `gnupg` is the only instance where the original code omitted `--no-install-recommends`, as far as I could find.

I checked why we're installing `gnupg`, and it's to allow us to install keys using `apt-key`. `apt-key` only requires `gnupg`, not the recommended packages of `gnupg`.

Lastly, to ensure this was true, I added an integ test for Debian 8, so now we have full integ coverage for the supported versions of Debian.

https://manpages.debian.org/stretch/apt/apt-key.8.en.html